enterprise/backend/src/metabase_enterprise/serialization/v2/load.clj

```diff
@@ -69,15 +73,29 @@
                                  :deps-chain expanding}
                                 e)))))))
 
+(defn- try-load-one
+  [ctx path]
+  (try
+    (load-one ctx path)
+    (catch Exception e
+      (log/info (trs "Error encountered loading {0}, continuing" (u.yaml/log-path-str path)))
+      (update ctx :errors conj e))))
+
 (defn load-metabase
   "Loads in a database export from an ingestion source, which is any Ingestable instance."
-  [ingestion]
+  [ingestion & {:keys [abort-on-error]}]
   ;; We proceed in the arbitrary order of ingest-list, deserializing all the files. Their declared dependencies guide
   ;; the import, and make sure all containers are imported before contents, etc.
   (serdes.backfill/backfill-ids)
-  (let [contents (serdes.ingest/ingest-list ingestion)]
-    (reduce load-one {:expanding #{}
-                      :seen      #{}
-                      :ingestion ingestion
-                      :from-ids  (m/index-by :id contents)}
-            contents)))
+  (let [paths    (serdes.ingest/ingest-list ingestion)
+        ctx      {:expanding #{}
+                  :seen      #{}
+                  :ingestion ingestion
+                  :from-ids  (m/index-by :id paths)
+                  :errors    []}
+        load-fn  (if abort-on-error load-one try-load-one)
+        result   (reduce load-fn ctx paths)]
+    (when-let [{:keys [errors]} result]
```

Comment:
could probably just destructure above but that would mess up the let alignment 😆